SECURITY.md

```diff
@@ -36,5 +36,14 @@ SOFTWARE.
 
 ## Reporting a Vulnerability
 
+**Please do not report any security vulnerabilities through GitHub issues.**
```

Comment:
Why not use GH's admonition system? Looks much better that way... See for ref: https://blog.solichain.com/how-to-create-admonitions-in-github-markdown-ab580debcfdc

Reply:
Excellent idea.